### PR TITLE
feat: enable stx tests

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of Leather — download [extension builds](https://github.com/leather-wallet/extension/actions/runs/6275824945).

See title

### CODE REVIEW

**Review by fbwoolf — APPROVED** [review]

**Review by kyranjamie — APPROVED** [review]

**Comment by alter-eggo** [comment]
cannot merge it until all checks pass, should be 27

**Comment by kyranjamie** [comment]
can you amend and force push, we've added billing now

**Comment by alter-eggo** [comment]
fixes: 
- mocked stx fees estimations (will put in separate file)
- remove scrollIntoViewIfNeeded as tests often fails bc of it, and not sure if we really need it
- put 3 testnet tests into 1 (each new test just start after click on back arrow and again on stx in list), which increased speed of test a lot (which is arguable as makes tests quite messy)

and seems now they work ok

**Comment by alter-eggo** [comment]
reworked stx send tests, most of them are on testnet and much faster now, as sign in happens only two times

File: tests/fixtures/fixtures.ts [hunk 1]
```diff
@@ -39,7 +39,6 @@ export const test = base.extend<TestFixtures>({
       ],
     });
     await use(context);
-    await context.close();
```
Comment (kyranjamie) [inline tests/fixtures/fixtures.ts:42]: sure we wanna remove this?

**Review by kyranjamie — APPROVED** [review]

File: tests/fixtures/fixtures.ts (hunk 1 above)
Comment (alter-eggo) [reply tests/fixtures/fixtures.ts:42]: with it that new stx tests not work

Comment (alter-eggo) [reply tests/fixtures/fixtures.ts:42]: it breaks nothing actually

**Pull request merged**